floodfill_selection/volume_selection.py

```diff
@@ -160,7 +159,7 @@ def __init__(self, keys='interactive'):
         self.volume.transform = scene.STTransform(translate=self.trans)
         self.view.add(self.volume)
 
-        self.tr = self.volume.node_transform(self.view)  # ChainTransform
+        self.tr = self.volume.get_transform(map_from='visual', map_to='canvas')
```

Comment:
I'm using the new transformation we use in the 3d viewers, since it allows the FOV to be any value